spec/features/18f_procurement_spec.rb

```diff
@@ -27,7 +27,7 @@
       expect(Dispatcher).to receive(:deliver_new_proposal_emails)
 
       visit '/gsa18f/procurements/new'
-      fill_in "Product Name and Description", with: "buying stuff"
+      fill_in "Product name and description", with: "buying stuff"
```

Comment:
No prob at all to punt - this was more of a general comment about what we might want to more toward in the future. I kind of hate I18n because the translation keys can get super long, but once you have it all set up it is nice to manage all app copy in one place....so something to think about